styleguide/derek/_partials/solutions/leadSplit.ejs

```diff
@@ -0,0 +1,13 @@
+<!-- Lead pattern -->
+<!--
+.lead class naming not used in alignment with typical pattern structure
+instead here .lead-wrapper is used to avoid having to override bootstrap base class
+ -->
+<div class="lead-wrapper">
+  <div class="lead-headerWrapper">
+    <h1 class="lead-header left">Title goes here</h1>
```

Comment:
Well thats the only way we would use it if its a lead left and something else right would be going into a subpanel well. I dont want to create a class that would only be used for zoolander documentation when we can just drop in the left class that does the same job. @WidgetsBurritos